registry/storage/driver/gcs/gcs.go

```diff
@@ -425,6 +426,10 @@ func (d *driver) putContent(ctx context.Context, obj *storage.ObjectHandle, cont
 	if _, err := bytes.NewReader(content).WriteTo(wc); err != nil {
 		return err
 	}
+	h := md5.New()
```

Comment:
Would it be worth adding a comment here (to save future visitors from having to go through hot history?

Reply:
LOL autocorrect; it's really `git` I meant, not `hot` (edited) 😆

Reply:
Makese sense: https://github.com/distribution/distribution/pull/4586/commits/7884c71297db23046eca9080479aad2c04c79375

Reply:
Thanks!